<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpuMemoryDump</class>
 <widget class="QDialog" name="GpuMemoryDump">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabbedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="generalTab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Memory Segment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadFromFilePushButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load from file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveToFilePushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save to file</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>115</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="bgTilesetCheckbox">
        <property name="text">
         <string>Background Tileset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="spTilesetCheckbox">
        <property name="text">
         <string>Sprites Tileset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tilemapsCheckbox">
        <property name="text">
         <string>Tilemaps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="oamCheckbox">
        <property name="text">
         <string>OAM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="paletteCheckbox">
        <property name="text">
         <string>Palette</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tilesetsTab">
    <attribute name="title">
     <string>Tilesets</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tilemapsTab">
    <attribute name="title">
     <string>Tilemaps</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="oamTab">
    <attribute name="title">
     <string>OAM</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="paletteTab">
    <attribute name="title">
     <string>Palette</string>
    </attribute>
    <widget class="QPushButton" name="dumpstdpalette">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dump standard palette</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Palette Viewer</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
